### PR TITLE
Fixes #19761 - revert use of new call for applicability

### PR DESCRIPTION
**Pull request by jlsherrill**
This reverts commit 7e9e55c61b77a81042dcf6362d641dfbe841df3f.

### CODE REVIEW

File: test/actions/pulp/consumer_test.rb [hunk 1]
```diff
@@ -8,13 +8,13 @@ class ConsumerTestBase < ActiveSupport::TestCase
     include VCR::TestCase
 
     let(:uuid) { 'uuid' }
-    let(:consumer_name) { 'gregor' }
+    let(:name) { 'name' }
```
Comment (beav) [inline test/actions/pulp/consumer_test.rb]: I would leave this change in. If you define a var named 'name', it will make the test auto-pass every time.

Comment (jlsherrill) [inline test/actions/pulp/consumer_test.rb]: but I 💓  auto-passing tests!

**Comment by beav** [comment]
APJ

**Comment by ehelms** [comment]
Test failure fun

**Pull request merged**